lib/lambda_vault_auth.rb

```diff
@@ -1,9 +1,6 @@
 require 'aws-sdk-core'
```

Comment:
We can remove the direct dep on aws-sdk-core now.

Reply:
We still need it for Aws::CredentialProviderChain, alas.